packages/liveblocks-node/src/Session.ts

```diff
@@ -1,14 +1,20 @@
 import type { AuthResponse } from "./client";
-import { assertNonEmpty, normalizeStatusCode } from "./utils";
+import {
+  assertNonEmpty,
+  normalizeStatusCode,
+  url,
+  URLSafeString,
+} from "./utils";
 
 // As defined in the source of truth in ApiScope in
 // https://github.com/liveblocks/liveblocks-cloudflare/blob/main/src/security.ts
 const ALL_PERMISSIONS = Object.freeze([
   "room:write",
   "room:read",
   "room:presence:write",
-  "comments:write",
-  "comments:read",
+  // TODO: Add these permissions back once the backend is set up to handle them
+  // "comments:write",
+  // "comments:read",
 ] as const);
```

Comment:
@nvie I have commented out `comments:write` and `comments:read` from `Sessions.ts` and added a comment to add the permissions back once the backend is set up to handle them.